src/mixins/communicationBridgeUtils.js

```diff
@@ -41,5 +41,11 @@ export default {
         },
       });
     },
+    /**
+     * @param {Object} data
+     */
+    handleContentUpdateFromBridge(data) {
```

Comment:
Right. Not calling `off` was causing lots of extra listeners, when navigating across pages, which hold references to the old View instances, which in the end was probably just piling up in memory 😨 

The other benefit of having it extracted, is that we can change the way we handle data, in one place.

I too am not a fan of the magic `topicData`, but I did not call it `updateTopicData` because that is too generic, and in the future, if we change the bridge API to use some more optimised format for example, it would be confusing.